source/getting_started/submit-patch.rst

```diff
@@ -0,0 +1,86 @@
+.. _submit-patch:
+
+Submit your patches
+===================
+
+|C| uses `GitHub Pull-Requests <https://help.github.com/articles/about-pull-requests/>`_ (PR) to submit and review code submissions. You are required to be aware of the software repository and branch in |C| source tree where your code is contirbuting before submitting any changes.
+
+|C| GitHub Manifest Structure
+-----------------------------
+
+|C| source code consists of multiple repositories ("**projects**"), the manifest file *.repo/manifests/default.xml* contains the information where ("**remotes**") to fetch those repositories. It includes the following files to instruct the ``repo`` command how to compose the |C| source tree:
+
+=============================  =============================================================
+include/aosp_vanilla.xml       This is vanilla AOSP code base on refs/tags/android-9.0.0_r18
+include/remove-android_ia.xml  The repositories to be removed from the AOSP code
+include/bsp-celadon.xml        The BSP source code customized for |C|
+=============================  =============================================================
+
+Use the following command to get the *remote* where you are trying to upload code:
+
+.. code-block:: bash
+
+   $ git remote -v
+
+The following table lists the *remote* aliases supported by |C|:
+
+=========  ========================================
+github     Good to go with github PR push process
+aosp       Create a google_diff patch & in-parallel
+           up-stream to aosp
+kernelorg  Upstream to kernel.org
+graphics   Upstream to graphics remote
+intel      Upstream to intel remote
+trusty-ia  Upstream to trusty-ia remote
+=========  ========================================
+
+You can further enter the following command to identify which branch the source head is pointing to:
```

Comment:
Suggested rewrite: 
Further, you can enter the following commands to identify which branch the source head points to:

Reply:
perfect!